list-of-macros.md

```diff
@@ -352,6 +352,7 @@ tests: [tests/test\_packages/test\_hyperref.py](tests/test_packages/test_hyperre
 **Macros**
 
 \\href,
+\\ref*,
```

Comment:
It is probably better to write `\*`, as '*' has special meaning in Markdown.

Reply:
Okay. This is the same for the `\cref*` commands.